packages/kbn-test/src/functional_tests/lib/run_kibana_server.ts

```diff
@@ -34,25 +34,19 @@ function extendNodeOptions(installDir?: string) {
   };
 }
 
-export async function runKibanaServer({
-  procs,
-  config,
-  options,
-  onEarlyExit,
-}: {
+export async function runKibanaServer(options: {
   procs: ProcRunner;
   config: Config;
-  options: {
-    installDir?: string;
-    extraKbnOpts?: string[];
-    logsDir?: string;
-  };
+  installDir?: string;
+  extraKbnOpts?: string[];
+  logsDir?: string;
   onEarlyExit?: (msg: string) => void;
 }) {
-  const runOptions = config.get('kbnTestServer.runOptions');
+  const { onEarlyExit, config, procs } = options;
```

Comment:
```suggestion
```
Looks like either obj destruction unnecessary since in the rest of the code it's `options.sth`

Reply:
It's unnecessary, but I think it's a lot clearer to read `config.get()` rather that `options.config.get()`, and there are enough calls to `config` and `procs` to justify destructuring those options. `onEarlyExit` is only used once though so I've removed that one.